protocol/v2/ssv/runner/runner_validations.go

```diff
@@ -96,6 +118,24 @@ func (b *BaseRunner) ValidatePostConsensusMsg(ctx context.Context, runner Runner
 
 		return b.verifyExpectedRoot(ctx, runner, psigMsgs, roots, domain)
 	}
+	if runner.GetRole() == spectypes.RoleCommittee {
+		validateMsg = func() error {
+			decidedValue := &spectypes.BeaconVote{}
+			if err := decidedValue.Decode(decidedValueBytes); err != nil {
+				return errors.Wrap(err, "failed to parse decided value to BeaconVote")
+			}
+
+			// Use b.State.CurrentDuty.DutySlot() since CurrentDuty never changes for CommitteeRunner
+			// by design, hence there is no need to store slot number on decidedValue for CommitteeRunner.
+			expectedSlot := b.State.CurrentDuty.DutySlot()
+			return b.validatePartialSigMsg(psigMsgs, expectedSlot)
+		}
+	}
+	if err := validateMsg(); err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
```suggestion
	return validateMsg()
```

or we can wrap the error